.github/workflows/go-provider.yml

```diff
@@ -1,8 +1,8 @@
 name: Provider Release
 on:
   push:
-#    tags:
-#      - 'v*'
+    tags:
+      - 'v*'
```

Comment:
I'd much like to see an [additional condition on the source of the tag, if possible](https://stackoverflow.com/a/78449380). 

WDYT?

Reply:
I.e. _the tag must have been created on the `main` branch_ and not in a random branch.